icu-date/ffi_cdef.lua

```diff
@@ -1,256 +1,259 @@
+
 return function(ffi, icu_version_suffix)
-  ffi.cdef([[
-  void* malloc(size_t size);
-  void free(void* ptr);
+  pcall(function ()
```

Comment:
You need to pcall of usage of defined/undefined function. Check your code youself, it does not work:

```
tarantool> ffi = require('ffi')
---
...

tarantool> pcall(ffi.cdef, 'int kill (int, int);')
---
- true
...

tarantool> pcall(ffi.cdef, 'int kill (int, int);')
---
- true
...
```

Also you often don't need to create an extra capture, say pcall(function() foo(x, y, z) end) can be written as pcall(foo, x, y, z).

Reply:
@Totktonada 

I had checked my code again. Look on this. When I trying to redefine `enum` `ffi.cdef` throws error

```
Tarantool 1.10.2-1-ge0017ad
type 'help' for interactive help
tarantool> ffi = require 'ffi'
---
...

tarantool> ffi.cdef('typedef enum UErrorCode { U_USING_FALLBACK_WARNING = -128 }')
---
...

tarantool> ffi.cdef('typedef enum UErrorCode { U_USING_FALLBACK_WARNING = -128 }')
---
- error: attempt to redefine 'UErrorCode'
...
```

Reply:
Yep, I don't take a type into consideration. Okay so.